pkg/environment/environment.go

```diff
@@ -58,13 +60,15 @@ var (
 	envDev = Environment{
 		RunningMode: RunningDev,
 		BcdbURL:     "https://explorer.devnet.grid.tf/explorer",
+		GraphQlURL:  "https://graphql.dev,grid.tf/graphql",
```

Comment:
```suggestion
		GraphQlURL:  "https://graphql.dev.grid.tf/graphql",
```

Reply:
Oh good catch, luckily we have no devnet anyway